docker-compose-deploy.yml

```diff
@@ -86,8 +91,9 @@ services:
       - REACT_APP_HTML_BODY_CLASS=${REACT_APP_HTML_BODY_CLASS}
       - REACT_APP_SENTRY_DSN=${REACT_APP_SENTRY_DSN}
       - REACT_APP_STRICT=${REACT_APP_STRICT}
+    restart: "no"
```

Comment:
Well done! When I first included restart rules in the former docker-deploy, I let the frontend restart itself, too. 😆